### PR TITLE
Merge branch and code cleaning

### PR DESCRIPTION
**Pull request by vinklein**
Remove some unused import

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 sudo: required
+group: deprecated-2017Q2
```
Comment (videlec) [inline .travis.yml]: why is that needed?

Comment (vinklein) [reply .travis.yml]: To keep working on the same travis trusty image as before the 19 june 2017.
I will try without this option to test if it's still needed.

Comment (vinklein) [reply .travis.yml]: Yep looks like it's still needed https://travis-ci.org/vinklein/pplpy/jobs/283175586

Comment (vinklein) [reply .travis.yml]: I have find a better workaround. Just use --user option on pip install. I will upload it asap
